cdap-ui/app/features/tracker/controllers/results-ctrl.js

```diff
@@ -150,6 +150,9 @@ class TrackerResultsController {
         createDate: entity.metadata.SYSTEM.properties['creation-time'],
         datasetType: entity.metadata.SYSTEM.properties.type
       });
+      if(entity.metadata.SYSTEM.tags.indexOf('explore') !== -1) {
```

Comment:
Just suggesting that the UI should be consistent in how it determines whether explore is enabled or not.